### PR TITLE
Made each faq question a class and placed the files in the proper folder layout. 

### PR DESCRIPTION
**Pull request by exploringsci**
I used the faq and faq-box from the 2020 fall site and changed some code to make sure the code fits the updated file/folder structure.

### CODE REVIEW

File: src/pages/faq.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
+  
 import React from 'react';
-import faqs from '../data/faqData.json';
+import faqs from '../components/faq.json';
```
Comment (DylanLi272) [inline src/pages/faq.js:3]: should be from `data` folder not `components`, should also be called `faqData.json`

File: src/pages/faq.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 import FAQBox from '../components/faq-box';
-import './faq.css';
 
 const FAQ = (props) => (
```
Comment (DylanLi272) [inline src/pages/faq.js:6]: this should be a class

File: src/components/faq.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+[
```
Comment (DylanLi272) [inline src/components/faq.json:1]: should delete this file

File: src/components/faq.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```
Comment (DylanLi272) [inline src/components/faq.js:1]: should delete this

File: src/components/faq.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.wrap{
```
Comment (DylanLi272) [inline src/components/faq.css:1]: should delete this

File: src/components/faq-box.js [hunk 6]
```diff
@@ -1,32 +1,24 @@
 import React, { Component } from 'react';
 import './faq-box.css';
 
-class FAQBox extends Component {
-    constructor(props) {
-        super(props);
+class FAQBox extends Component {
```
Comment (DylanLi272) [inline src/components/faq-box.js:4]: use the prettier thing to format this

File: src/components/faq-box.css [hunk 7]
```diff
@@ -1,27 +1,24 @@
...
+ }
+ 
+ .question {
     font-size: 1.3em;
-}
-
-.answer {
-    font-family: 'Arial, Helvetica, sans-serif';
-}
+ }
```
Comment (DylanLi272) [inline src/components/faq-box.css:20]: use prettier thing to format this

**Review by DylanLi272 — CHANGES_REQUESTED** [review]
I made comments in specific files, please make those changes and make a new pr

**Comment by exploringsci** [comment]
Thanks for the requested changes. I made the changes. I'll close this pull review and create a new pr that fixed the issues you pointed out. Thanks!

**Pull request closed without merging**